src/operator/webhooks/clientintents_webhook_v1.go

```diff
@@ -178,7 +178,7 @@ func (v *IntentsValidatorV1) validateSpec(intents *otterizev1beta1.ClientIntents
 			}
 			for _, dns := range intent.Internet.Domains {
 				_, err := idna.Lookup.ToASCII(dns)
-				if err != nil {
+				if err != nil && !strings.Contains(dns, "*") {
```

Comment:
check for `strings.HasPrefix("*")`. We wouldn't want to support starts in the middle of the domain name.